js/client/modules/@arangodb/testsuites/arangosh.js

```diff
@@ -125,7 +125,7 @@ function arangosh (options) {
       ret[section].failed = 1;
       ret[section]['message'] =
         'didn\'t get expected return code (' + expectedReturnCode + '): \n' +
-        yaml.safeDump(rc);
+        yaml.dump(rc);
```

Comment:
This method was renamed. I'm not sure if we consider `js-yaml` a publicly supported bundled module. If so, this is a breaking change and probably needs to be rolled back?

Reply:
Our documentation says that is a bundled (supported) npm module: 
- https://www.arangodb.com/docs/3.7/appendix-java-script-modules.html#bundled-npm-modules

Reply:
Okay, I'll roll this one back then.

Reply:
Done.